assets/app/views/projects.html

```diff
@@ -1,18 +1,28 @@
 <div class="container">
-  <h1>Projects</h1>
-  <div class="tile tile-project tile-click" style="margin-bottom: 10px;" ng-repeat="project in projects | orderByDisplayName">
+  <div>
+    <h1 style="display: inline-block;">Projects</h1>
+    <a ng-if="canCreate" href="createProject" style="margin-top: 10px;" class="btn btn-lg btn-primary pull-right">
+      <span>Create</span>
+      <i class="fa fa-plus" />
+    </a>
+  </div>
+  <div class="tile tile-project tile-click" style="margin: 10px 0;" ng-repeat="project in projects | orderByDisplayName">
     <h2 class="project">
       <a class="tile-target" href="project/{{project.metadata.name}}">{{(project | displayName) || project.metadata.name}}</a>
       <span  ng-if="project.status.phase != 'Active'" class="pficon-layered" data-toggle="tooltip" data-placement="right" title="This project has been marked for deletion." style="cursor: help; vertical-align: top; margin-left: 5px;">
         <span class="pficon pficon-warning-triangle"></span>
         <span class="pficon pficon-warning-exclamation"></span>
-      </span>      
+      </span>
     </h2>
     <div class="muted" style="margin-top: -5px;" ng-if="project | annotation : 'description'">{{project | annotation : 'description'}}</div>
   </div>
   <div ng-if="emptyMessage && (projects | hashSize) == 0">{{emptyMessage}}</div>
-  <div style="margin-top: 10px;">
-    To create a new project, run <code>osc new-project &lt;projectname&gt;</code>
+  <div ng-if="canCreate === false" style="margin-top: 10px;">
+    <span ng-if="!newProjectMessage">An administrator can create a project for you
+      by running the command
+      <code>osadm new-project &lt;projectname&gt;
+        --admin={{user.metadata.name || '&lt;YourUsername&gt;'}}</code></span>
```

Comment:
No :)

![2__vim__vim_](https://cloud.githubusercontent.com/assets/1167259/7659641/763bb88e-fb11-11e4-8576-c0fc4596453c.png)

But I'll change it.